sample-apps/react-native/dogfood/src/components/ActiveCall.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useRef } from 'react';
+import {
+  CallControlsView,
+  CallParticipantsBadge,
+  CallParticipantsView,
+  CallingState,
+  useCall,
+  useIncallManager,
+} from '@stream-io/video-react-native-sdk';
+import { ActivityIndicator, SafeAreaView, StyleSheet } from 'react-native';
+import { appTheme } from '../theme';
+import { useChannelWatch } from '../hooks/useChannelWatch';
+import { useUnreadCount } from '../hooks/useUnreadCount';
+
+type ActiveCallProps = {
+  chatButton?: {
+    onPressHandler: () => void;
+  };
+};
+
+export const ActiveCall = ({ chatButton }: ActiveCallProps) => {
+  const channelWatched = useChannelWatch();
+  const unreadBadgeCountIndicator = useUnreadCount({ channelWatched });
+
+  const call = useCall();
+  const activeCallRef = useRef(call);
+  activeCallRef.current = call;
+
+  useEffect(() => {
+    return () => {
+      if (activeCallRef.current?.state.callingState !== CallingState.LEFT) {
+        activeCallRef.current?.leave();
+      }
+    };
+  }, []);
+
+  useIncallManager({ media: 'video', auto: true });
```

Comment:
Can you add an explanation why we're calling this hook, it will help to be in parity with the tutorial?